packages/protocol-http/src/httpHandler.ts

```diff
@@ -6,7 +6,7 @@ import { HttpResponse } from "./httpResponse";
 /**
  * @internal
  */
-export type HttpHandler<HttpHandlerConfig extends object = Record<string, unknown>> = RequestHandler<
+export type HttpHandler<HttpHandlerConfig extends object = object> = RequestHandler<
```

Comment:
use `X extends object = {}`

Reply:
Is using `object` the same as `{}`? Isn't `{}` a literal type?